### PR TITLE
Feature/table cells

### PR DESCRIPTION
**Pull request by CrazyCrud**
The PAGE-XML functionality has been extented in order to be able to create TableCell elements.
Also an error was fixed which occured when trying to transform a list to points.

### CODE REVIEW

File: dh_segment/io/PAGE.py [hunk 1]
```diff
@@ -400,6 +435,82 @@ def from_dict(cls, dictionary: dict) -> 'TextRegion':
...
+        cell_et.set('embText', _encode_bool(self.embedded_text))
+        return cell_et
+
+    @classmethod
+    def from_xml(cls, e: ET.Element) -> 'TableCell':
+        cls.check_tag(e.tag)
+        return TableCell(
+            **super().from_xml(e),
+            row=int(e.attrib.get('row')),
+            col=int(e.attrib.get('col')),
```
Comment (solivr) [inline dh_segment/io/PAGE.py]: isn't it 'rowIndex' and 'columnIndex' ?

File: dh_segment/io/PAGE.py [hunk 2]
```diff
@@ -400,6 +435,82 @@ def from_dict(cls, dictionary: dict) -> 'TextRegion':
...
+        self.row = row
+        self.col = col
+        self.row_span = row_span
+        self.col_span = col_span
+        self.embedded_text = embedded_text if embedded_text is not None else False
+
+    def to_xml(self, name_element='TableCell') -> ET.Element:
+        cell_et = super().to_xml(name_element=name_element)
+        if self.row is not None:
+            cell_et.set('row', '{}'.format(self.row))
```
Comment (solivr) [inline dh_segment/io/PAGE.py]: isn't it 'rowIndex' ?

File: dh_segment/io/PAGE.py [hunk 3]
```diff
@@ -400,6 +435,82 @@ def from_dict(cls, dictionary: dict) -> 'TextRegion':
...
+        if self.col is not None:
+            cell_et.set('col', '{}'.format(self.col))
```
Comment (solivr) [inline dh_segment/io/PAGE.py]: isn't it 'columnIndex' ?

**Comment by solivr** [comment]
Hi @CrazyCrud,

Thanks a lot for your PR. I've added some formatting to the init functions of the classes and also changed the formatting of the strings to be compatible with python 3.5.

Before merging, could you check that the TableCell element has the proper attributes names (with respect to this schema: https://www.primaresearch.org/schema/PAGE/gts/pagecontent/2019-07-15/pagecontent.xsd)? Also according to this schema "header" attribute seems to be missing.

**Comment by CrazyCrud** [comment]
Hey @solivr,

No problem, always happy to help.

Thank you for your notes. I tried to correct the attributes and hope it corresponds to the PAGE schema.

**Comment by solivr** [comment]
Thanks, I've added your changes to the `master` branch.

**Pull request closed without merging**